04_containers.Rmd

```diff
@@ -766,10 +733,18 @@ tse
 
 Now `rowTree` includes a phylogenetic tree:
 
-```{r, eval=FALSE}
+```{r, eval = FALSE}
 head(rowTree(tse))
 ```
 
+Finally, we save the complete TreeSE object as an RDS file, so that we can
+readily import and use it in the future.
+
+```{r, eval = FALSE}
+# SAVE as RDS file
+saveRDS(tse, file = "tse.rds")
+```
```

Comment:
Do we really need to show this, unless we ourselves need it for book examples (which is not the case here with eval=FALSE). -> I suggest to remove lines 740-746

Reply:
This is sort of standard R knowledge, we don't have to teach everything.